src/main/java/com/twilio/http/ValidationClient.java

```diff
@@ -104,7 +107,14 @@ public Response makeRequest(Request request) {
 
         HttpMethod method = request.getMethod();
         if (method == HttpMethod.POST) {
-            builder.addHeader(HttpHeaders.CONTENT_TYPE, "application/x-www-form-urlencoded");
+            if(request.getContentType() != null && request.getContentType().equals("APPLICATION_JSON")){
```

Comment:
Why is the content type the string literal "APPLICATION_JSON".  Shouldn't it be "application/json"?

Reply:
Yes